### PR TITLE
Involved changes to nat.examples

### PR DESCRIPTION
**Pull request by alexanderbates**
* Wide-ranging changes to the examples in nat.examples
* I think it might be good to merge these changes in now in bulk because as Sri converts them to Jupyter notebooks he'll also be checking them

### CODE REVIEW

File: 05-miyasaka2014/06-nblast-masked.R [hunk 1]
```diff
@@ -1,42 +1,82 @@
...
 zm_dend <- as.dendrogram(zm_clust)
-
 zm_cols <- c(maG="#3635fd", dG="#f08380", vaG="#df20be", lG="#6e9d3f", vpG="#6a6d26", mdGT="#87b3d8", mdGL="#dea76f", vmG="#454545")
 par(mar=c(3,1,1,5))
 zm_dend_col=set_leaf_colors(zm_dend, structure(zm_cols[zm_class],.Names=names(zm)), col_to_set='label')
 zm_dend_col=color_clusters(zm_dend_col,k=4,col=rep('black',4),groupLabels=as.roman)
 plot(zm_dend_col, horiz=TRUE)
 
+# NBlast more neatly
+remove = sapply(zm.mask.vTel, function(x) nrow(xyzmatrix(x))>1);sum(remove) # Remove too-small neuron
```
Comment (jefferis) [inline 05-miyasaka2014/06-nblast-masked.R]: For future ref `nat::nvertices()` works well for this.

File: 03-helmstaedter2013/00-setup.R [hunk 2]
```diff
@@ -1,6 +1,20 @@
 if(!require('nat')) install.packages("nat")
+if(!require('nat.nblast')) install.packages("nat.nblast")
+if(!require('dendextend')) install.packages("dendextend")
+if(!require('R.matlab')) install.packages("R.matlab")
+if(!require('dendroextras')) install.packages("dendroextras")
```
Comment (jefferis) [inline 03-helmstaedter2013/00-setup.R]: installed by `nat.nblast`

File: 03-helmstaedter2013/00-setup.R [hunk 3]
```diff
@@ -1,6 +1,20 @@
 if(!require('nat')) install.packages("nat")
```
Comment (jefferis) [inline 03-helmstaedter2013/00-setup.R]: installed by `nat.nblast`

File: 02-grosjean2011/01-download-plot3d.R [hunk 4]
```diff
@@ -16,8 +16,9 @@ open3d()
 plot3d(MyNeurons[[1]])
 
 # Clear 3d display and plot all neurons
-clear3d()
 plot3d(MyNeurons,WithNode=FALSE)
+# Take a picture!
+rgl.snapshot(filename ="images/nat_Grosjean_Olfactory_PNs.png" ,fmt = "png")
```
Comment (jefferis) [inline 02-grosjean2011/01-download-plot3d.R]: note that this will fail if the images subfolder doesn't exist.

**Comment by alexanderbates** [comment]
Okay, I shall do that!

**Pull request closed without merging**